### PR TITLE
opencontrail: fix build when false

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

**Comment by lebauce** [comment]
I pushed a branch (https://github.com/skydive-project/skydive/pull/1226) last week to fix this problem but I completely forgot to create a PR for it, sorry.

**Comment by hunchback** [comment]
:-)

**Comment by hunchback** [comment]
@lebauce - I reopened as when opencontrail=false there is no reason to build the easyjson for opencontrail which is time consuming.

File: Makefile [hunk 1]
```diff
@@ -224,8 +224,12 @@ debug.analyzer:
 	$(call VENDOR_RUN,${EASYJSON_GITHUB}) easyjson -build_tags linux ${EASYJSON_FILES_TAG_LINUX}
 
 .PHONY: .easyjson.tag.opencontrail
+ifeq ($(WITH_OPENCONTRAIL), true)
```
Comment (nplanel) [inline Makefile]: please put the if around the call not the makefile rule

**Comment by lebauce** [comment]
@hunchback I completely aggre. I'm working on a patch that call easyjson only when required. Closing this one for now.

**Comment by hunchback** [comment]
Fine

**Pull request closed without merging**